### PR TITLE
Local storage feature

### PR DESCRIPTION
**Pull request by 10-menachi**
Added:
-  Feature to store user data to local storage
-  Feature to retrieve user data from local storage

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -194,3 +194,27 @@ form.addEventListener('submit', (e) => {
...
+const saveDataToStorage = () => {
+  const data = {
+    name: name.value,
+    email: email.value,
+    message: message.value,
+  };
+  localStorage.setItem('data', JSON.stringify(data));
+};
+window.addEventListener('load', getDataFromStorage);
+form.addEventListener('submit', saveDataToStorage);
```
Comment (Strangeal) [inline index.js]: Great work so far! As specified in the project requirements, user input should be `stored` in the `localStorage` whenever they input or change data in the input field. Your current code works well, but it only stores the user's data in the `localStorage` when they hit the `submit` button. However, this is not the intended behavior. Please modify your code to store the user's data in the `localStorage` whenever they input or change data in the input field. Thank you!
Screenshot 2023-04-15 at 08 43 52

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @10-menachi and @iamsjunaid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (10-menachi) [reply index.js]: Thank you for the review

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @10-menachi and @iamsjunaid , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All requested changes implemented ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**